eventfiles/static/event_files/delete_pop_up.css

```
@@ -0,0 +1,68 @@
+a:link { text-decoration: none; }
+
+a:visited { text-decoration: none; }
+
+a:hover { text-decoration: none; }
+
+a:active { text-decoration: none; }
+
+
+#dialogoverlay{
+	display: none;
+	opacity: .8;
+	position: fixed;
+	top: 0px;
+	left: 0px;
+	background: #FFF;
+	width: 100%;
+	z-index: 10;
+}
+#dialogbox{
+    z-index:99999;
+	display: none;
+	position: absolute;
+	background: #000;
+	border-radius:7px;
+	width:550px;
+	z-index: 10;
+	top: 50%;
+	left 50%;
```

Maybe you missed colon after `left`?
```suggestion
	left: 50%;
```

---

Done